### ISSUE TITLE
Error on CLI: `Unknown argument: src/index.ts`

### ISSUE THREAD

**Issue by martinfrancois** [acrazing/dpdm#15]
To reproduce in a TypeScript project, create a `src` folder, inside of that folder create a file called `index.ts`, then run:
```
npm i -g dpdm
dpdm src/index.ts
```
With version 3.6.0 or earlier, the output is correct and as expected (shows no circular dependencies found).

However with version 3.6.1 and 3.7.0 the output shows:
```
Unknown argument: src/index.ts
```

If I first run `cd src` and then run `dpdm index.ts` it's the same result.

Can reproduce on both macOS and Ubuntu.

**Comment by acrazing** [comment]
Thanks, it is caused by `yargs@17.1.0` and fixed in `dpdm@3.7.1`.

**Comment by martinfrancois** [comment]
Thanks for fixing this so quickly! I can confirm it indeed resolved the issue with 3.7.1.
